modules/ROOT/pages/index.adoc

```diff
@@ -95,7 +220,7 @@ include::reuse::partial$billing/pricing.adoc[tag=monitoringTable]
 [[plans-legacy]]
 === Anypoint Monitoring Features for Legacy Subscription-Based Plans
 
-Anypoint Monitoring is the standard method of monitoring Mule application and API performance so that you can more quickly identify and resolve issues. All organizations receive fundamental monitoring capabilities with their subscriptions. However, customers subscribing at the Titanium tier receive greater storage, guaranteed resources, and advanced capabilities that allow for greater control and flexibility.
+Anypoint Monitoring is the standard method of monitoring Mule app and API performance so that you can more quickly identify and resolve issues. All organizations receive fundamental monitoring capabilities with their subscriptions. However, customers subscribing at the Titanium tier receive greater storage, guaranteed resources, and advanced capabilities that allow for greater control and flexibility.
```

Comment:
What's the diff between Integration Advanced and Titanium?

Reply:
Core based and Usage based subscription features have different audiences. There are 5 tiers right now with some differences. As things settle over time, we'll make adjustments. Right now, we'll keep as-is.